### PR TITLE
Refine description for Enable-AzSynapseWorkspace cmdlet help doc

### PR DESCRIPTION
**Pull request by yanjungao718**
## Description

Refine description for Enable-AzSynapseWorkspace cmdlet help doc

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by yanjungao718** [comment]
@idear1203 Dongwei, please help to review the help doc, thanks!

File: src/Synapse/Synapse/help/Enable-AzSynapseWorkspace.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ schema: 2.0.0
 # Enable-AzSynapseWorkspace
 
 ## SYNOPSIS
-Activate a new Azure Synapse Analytics workspace.
+When creating an Azure Synapse Analytics workspace, you can choose to encrypt all data at rest in the workspace 
+with a customer-managed key which will provide double encryption to the workspace.
+You may need to set up the encryption environment firstly, such as to create a key vault with purge protection enable and 
+specify Access Polices to the key vault.
+Then use this cmdlet to activate the new Azure Synapse Analytics workspace which double encryption is enabled using a customer-managed key.
```
Comment (idear1203) [inline src/Synapse/Synapse/help/Enable-AzSynapseWorkspace.md:15]: Put the description in one line? The file will be eventually transformed to a web page of Microsoft Docs. I suppose the website can handle the word wrapping.

Comment (idear1203) [reply src/Synapse/Synapse/help/Enable-AzSynapseWorkspace.md:15]: The description itself looks great to me.

**Review by idear1203 — DISMISSED** [review]

File: src/Synapse/Synapse/help/Enable-AzSynapseWorkspace.md (hunk 1 above)
Comment (yanjungao718) [reply src/Synapse/Synapse/help/Enable-AzSynapseWorkspace.md:15]: Sure, I will put the words in one line.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**